includes/class-sensei-usage-tracking-data.php

```diff
@@ -23,6 +23,10 @@ class Sensei_Usage_Tracking_Data {
 	public static function get_usage_data() {
 		return array(
 			'courses' => wp_count_posts( 'course' )->publish,
+			'courses_with_video' => self::get_courses_with_video_count(),
+			'courses_with_disabled_notification' => self::get_courses_with_disabled_notification_count(),
```

Comment:
Is it possible to shorten these keys a bit? Maybe `course_videos`, `course_no_notifications` and `course_prereqs`? I'm hoping to avoid us hitting the query parameter limit.

Reply:
Changed